src/common/sound.cpp

```diff
@@ -1,8 +1,9 @@
-#include "sound_C_wrapper.h"
-#include "sound.h"
+#include "sound.hpp"
 #include "hwio.h"
 #include "eeprom.h"
 
+uint8_t SOUND_INIT = 0;
```

Comment:
make it static to be visible only in this file
use enum class instead

Reply:
or bool if it is only true/false

Reply:
changed to - static bool